e2e/9_swaps.spec.ts

```diff
@@ -62,9 +62,7 @@ describe('Swap Sheet Interaction Flow', () => {
     await tap('swap-button');
     await delayTime('very-long');
 
-    // flaky
-    // await swipeUntilVisible('token-to-buy-dai-1', 'token-to-buy-list', 'up', 100);
-    await swipe('token-to-buy-list', 'up', 'slow', 0.2);
+    await swipeUntilVisible('token-to-buy-dai-1', 'token-to-buy-list', 'up', 100);
```

Comment:
did you verify this works as expected now?

Reply:
all the runs before thsi tests were failing. This is passing, so I'd say yeah